java/arcs/android/storage/service/StorageServiceManager.kt

```diff
@@ -36,7 +37,7 @@ class StorageServiceManager(
     private val coroutineContext = parentCoroutineContext + CoroutineName("StorageServiceManager")
 
     override fun clearAll(resultCallback: IResultCallback) {
-        runBlocking(coroutineContext) {
+        CoroutineScope(coroutineContext).launch {
```

Comment:
should we just create one `val scope = CoroutineScope(coroutineContext)` and just use `scope.launch`? (Not sure how much it matters, but it does give us an owned scope to eventually cancel somehow if needed)

Reply:
yeah, that's a good plan.